### DISCUSSION TITLE
where's fake_1000_labels.csv?

### DISCUSSION THREAD

**Discussion by staropram** [moj-analytical-services/splink#1303] [Q&A] [answered]
I've gone through the tutorial. Great job btw.

I couldn't find fake_1000.csv so I used [fake_1000_from_splink_demos.csv](https://github.com/moj-analytical-services/splink/blob/master/tests/datasets/fake_1000_from_splink_demos.csv). This actually doesn't work with the tutorial as the m vector fails to converge due to the dob comparison rule. This was good for me because I used it as an opportunity to learn how to change the comparison rules to get it to work. You might want to update the tutorial to point to the fake_1000.csv dataset if it exists.

The "assurances" part of the tutorial refers to "fake_1000_labels.csv", but I can't find this, where is it?

Cheers

**Comment by RobinL** [comment, marked as answer]
All the tutorial data and code is in the splink_demos repo - the specific file is here:
https://github.com/moj-analytical-services/splink_demos/blob/d56a2404399b16697af174f39974d7f522cd9ed5/data/fake_1000.csv
Sorry - this should probably be signposted better
